src/sql/parts/notebook/models/notebookModel.ts

```diff
@@ -71,7 +74,7 @@ export class NotebookModel extends Disposable implements INotebookModel {
 	private _kernelDisplayNameToNotebookProviderIds: Map<string, string> = new Map<string, string>();
 	private _onValidConnectionSelected = new Emitter<boolean>();
 
-	constructor(private _notebookOptions: INotebookModelOptions, startSessionImmediately?: boolean, private connectionProfile?: IConnectionProfile) {
+	constructor(private _notebookOptions: INotebookModelOptions, private _notebookParams?: INotebookParams, startSessionImmediately?: boolean, private connectionProfile?: IConnectionProfile) {
```

Comment:
As mentioned offline let's refactor so we don't need to reference the editor model at all. We'll define a new IContentManager that serves the correct purpose here instead.